Sources/PaymentsCore/Networking/APIClient.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+public class APIClient {
+    public typealias CorrelationId = String
```

Comment:
`CorrelationId` > `CorrelationID` to match the ID name-spacing used in other parts of the SDK like for `clientID`?